### PR TITLE
Implement junit tests for ContactListParser class

### LINKED ISSUE #620: Test: Implement Junit tests for ContactListParser

### PR DESCRIPTION
**Pull request by jonasngs**
This PR fixes #620

### CODE REVIEW

File: src/test/java/seedu/address/testutil/PersonUtil.java [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
+import java.util.Set;
+
+import seedu.address.logic.commands.contactlistcommands.AddContactCommand;
+import seedu.address.logic.commands.contactlistcommands.EditContactDescriptor;
 import seedu.address.model.contact.Contact;
-//import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.Tag;
 
 /**
  * A utility class for Person.
```
Comment (michael-setia) [inline src/test/java/seedu/address/testutil/PersonUtil.java:16]: I think we can change `Person` to `Contact` if time allows.

**Review by michael-setia — APPROVED** [review]
LGTM.

File: src/test/java/seedu/address/testutil/PersonUtil.java (hunk 1 above)
Comment (murtubak) [inline src/test/java/seedu/address/testutil/PersonUtil.java:16]: Perhaps should change from person to contact for clarity.

**Review by murtubak — APPROVED** [review]
LGTM. Minor documentation issue to clarify.

File: src/test/java/seedu/address/testutil/PersonUtil.java (hunk 1 above)
Comment (jonasngs) [reply src/test/java/seedu/address/testutil/PersonUtil.java:16]: Thanks for the note, this was actually resolved in PR #598

Comment (jonasngs) [reply src/test/java/seedu/address/testutil/PersonUtil.java:16]: Thanks for the note, this was resolve in PR #598.

**Pull request merged**